[2023-04-14 17:17:41,739] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: vic_investigacion_ETL.get_cv_lac_data manual__2023-04-14T17:17:40.547206+00:00 [queued]>
[2023-04-14 17:17:41,748] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: vic_investigacion_ETL.get_cv_lac_data manual__2023-04-14T17:17:40.547206+00:00 [queued]>
[2023-04-14 17:17:41,749] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-04-14 17:17:41,750] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2023-04-14 17:17:41,750] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-04-14 17:17:41,759] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): get_cv_lac_data> on 2023-04-14 17:17:40.547206+00:00
[2023-04-14 17:17:41,766] {standard_task_runner.py:52} INFO - Started process 1113 to run task
[2023-04-14 17:17:41,771] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'vic_investigacion_ETL', 'get_cv_lac_data', 'manual__2023-04-14T17:17:40.547206+00:00', '--job-id', '117', '--raw', '--subdir', 'DAGS_FOLDER/vic_investigacion_dag.py', '--cfg-path', '/tmp/tmp7ytoxyt1', '--error-file', '/tmp/tmpn0hgjf_z']
[2023-04-14 17:17:41,774] {standard_task_runner.py:80} INFO - Job 117: Subtask get_cv_lac_data
[2023-04-14 17:17:41,856] {task_command.py:371} INFO - Running <TaskInstance: vic_investigacion_ETL.get_cv_lac_data manual__2023-04-14T17:17:40.547206+00:00 [running]> on host d014c8b814c7
[2023-04-14 17:17:41,923] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=vic_investigacion_ETL
AIRFLOW_CTX_TASK_ID=get_cv_lac_data
AIRFLOW_CTX_EXECUTION_DATE=2023-04-14T17:17:40.547206+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-14T17:17:40.547206+00:00
[2023-04-14 17:17:41,924] {logging_mixin.py:115} INFO - Getting CV_LAC Data
[2023-04-14 17:17:41,925] {python.py:173} INFO - Done. Returned value was: None
[2023-04-14 17:17:41,939] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=vic_investigacion_ETL, task_id=get_cv_lac_data, execution_date=20230414T171740, start_date=20230414T171741, end_date=20230414T171741
[2023-04-14 17:17:41,985] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-04-14 17:17:42,024] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
